src/script/view_model/ContentViewModel.ts

```diff
@@ -150,7 +150,7 @@ export class ContentViewModel {
       return conversationEntity;
     }
 
-    return this.conversationRepository.init1to1Conversation(conversationEntity);
+    return this.conversationRepository.init1to1Conversation(conversationEntity, true);
```

Comment:
Maybe we could do that only if we are in an MLS env, would that make sense?

Reply:
This functionality should work fine even if MLS is disabled on backend, so there's no need to perform any mls support check here ;)